Gemfile.lock

```diff
@@ -400,7 +400,7 @@ GEM
       rails-dom-testing (>= 1, < 3)
       railties (>= 4.2.0)
       thor (>= 0.14, < 2.0)
-    json (2.5.0)
+    json (2.5.1)
```

Comment:
dependency of recaptcha, bug fixes: https://github.com/flori/json/blob/master/CHANGES.md